### ISSUE TITLE
Support `/video/*`

### ISSUE THREAD

**Issue by DUOLabs333** [libreddit/libreddit#368]
## How does this feature work on Reddit?

`reddit.com/video/aaabbcc` will link to a video
## Describe the implementation into Libreddit

To do the exact same thing.

## Describe alternatives you've considered

`/video` is a simple redirect to the post, so it's just a matter for finding the post that corresponds to the video URL.
## Additional context

Example: reddit.com/video/4j3ovwqpzy581

**Comment by DUOLabs333** [comment]
It seems to work already.

**Comment by Cynosphere** [comment]
Does not work (anymore?) on v0.23.2. Tried on two separate instances (`discuss.whatever.social` and `libreddit.pussthecat.org`)

**Comment by spikecodes** [comment]
Yeah, I don't believe this feature is already implemented so I'll reopen this issue.
